From 6682471436fbe63770d5a6d3958dccd770622592 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jose.dapena@lge.com>
Date: Thu, 14 Feb 2019 15:45:22 +0100
Subject: IWYU: ExtensionRegistry in BookmarkAppTabHelper.

Fixes GCC build.

Change-Id: Icf452ceceb128dadce34c8aa6c465f085b61c25c

diff --git a/chrome/browser/web_applications/extensions/bookmark_app_tab_helper.h b/chrome/browser/web_applications/extensions/bookmark_app_tab_helper.h
index 895e204734ec..11dfdefa7e81 100644
--- a/chrome/browser/web_applications/extensions/bookmark_app_tab_helper.h
+++ b/chrome/browser/web_applications/extensions/bookmark_app_tab_helper.h
@@ -8,6 +8,7 @@
 #include "base/macros.h"
 #include "base/scoped_observer.h"
 #include "chrome/browser/web_applications/components/web_app_tab_helper_base.h"
+#include "extensions/browser/extension_registry.h"
 #include "extensions/browser/extension_registry_observer.h"
 
 namespace content {
@@ -17,7 +18,6 @@ class WebContents;
 namespace extensions {
 
 class Extension;
-class ExtensionRegistry;
 
 // Allows to associate a tab with bookmark app.
 class BookmarkAppTabHelper : public web_app::WebAppTabHelperBase,
-- 
2.17.1

