From eefb02919ceefaccc3a2197c5caa845416792442 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Tue, 14 Jul 2020 17:39:11 +0200
Subject: IWYU: memcpy required cstring in byte_array.h

Bug: chromium:819294
---
 cpp/platform_v2/base/byte_array.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cpp/platform_v2/base/byte_array.h b/cpp/platform_v2/base/byte_array.h
index ee5d0eb..4b1d79b 100644
--- a/cpp/platform_v2/base/byte_array.h
+++ b/cpp/platform_v2/base/byte_array.h
@@ -17,6 +17,7 @@
 
 #include <array>
 #include <cstdint>
+#include <cstring>
 #include <string>
 #include <type_traits>
 #include <utility>
-- 
2.25.1

