From 4826ec987145d7d3f0bce48261d6ff67f6dfe568 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jose.dapena@lge.com>
Date: Thu, 4 Apr 2019 22:37:40 +0200
Subject: LanguagePrefs: std::set of const is forbidden in libstdc++.

Change-Id: I018a92ae0f5caaa8e369ffbae34eb294f9aefb39

diff --git a/components/language/core/browser/language_prefs.cc b/components/language/core/browser/language_prefs.cc
index 317aa1d8298c..df258ca62acf 100644
--- a/components/language/core/browser/language_prefs.cc
+++ b/components/language/core/browser/language_prefs.cc
@@ -85,7 +85,7 @@ void LanguagePrefs::ResetEmptyFluentLanguagesToDefault() {
 }
 
 base::Value LanguagePrefs::GetDefaultFluentLanguages() {
-  std::set<const std::string> languages;
+  std::set<std::string> languages;
 #if defined(OS_CHROMEOS)
   // Preferred languages.
   std::string language = language::kFallbackInputMethodLocale;
-- 
2.19.1

