From bba49bf773198ed24e0fc85989a591e6fc0a5e3a Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jose.dapena@lge.com>
Date: Fri, 25 Oct 2019 17:45:56 +0200
Subject: [PATCH] GCC: remove BlinkCloneableMessage noexcept in move assignment
 operator.

Bug: 819294
Change-Id: Ibcb9fd4cfff87c8f2a044ff13c55df155aac4308

diff --git a/third_party/blink/renderer/core/messaging/blink_cloneable_message.cc b/third_party/blink/renderer/core/messaging/blink_cloneable_message.cc
index b6c876841587..129ceda13d2f 100644
--- a/third_party/blink/renderer/core/messaging/blink_cloneable_message.cc
+++ b/third_party/blink/renderer/core/messaging/blink_cloneable_message.cc
@@ -12,6 +12,6 @@ BlinkCloneableMessage::~BlinkCloneableMessage() = default;
 BlinkCloneableMessage::BlinkCloneableMessage(BlinkCloneableMessage&&) noexcept =
     default;
 BlinkCloneableMessage& BlinkCloneableMessage::operator=(
-    BlinkCloneableMessage&&) noexcept = default;
+    BlinkCloneableMessage&&) = default;
 
 }  // namespace blink
-- 
2.20.1

