diff --git a/tools/cddl/cddl.py b/tools/cddl/cddl.py
index 5ad9c90..7c1129e 100644
--- a/tools/cddl/cddl.py
+++ b/tools/cddl/cddl.py
@@ -32,7 +32,7 @@ def main():
 
   if (args.verbose):
     print('Creating C++ files from provided CDDL file...')
-  echoAndRunCommand(['./cddl', "--header", args.header, "--cc", args.cc,
+  echoAndRunCommand(['./v8-qemu-wrapper.sh', './cddl', "--header", args.header, "--cc", args.cc,
                      "--gen-dir", args.gen_dir, args.file],
                      False, log, args.verbose)
 
